src/importNameRule.ts

```diff
@@ -45,15 +45,15 @@ export class Rule extends Lint.Rules.AbstractRule {
 
         if (options.ruleArguments instanceof Array) {
             options.ruleArguments.forEach((opt: unknown, index: number) => {
-                if (index === 1 && isObject(opt)) {
+                if (index === 0 && isObject(opt)) {
```

Comment:
This will be a breaking change. I thought that this might be side effect of migration to new rule format, however this behavior was in `6.0.0`.
Looks like better solution is needed to avoid breaking change for those who added one more element to options array to match expected indices